pymc_experimental/tests/statespace/test_VARMAX.py

```diff
@@ -0,0 +1,101 @@
+import os
+import sys
+import warnings
+from itertools import product
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pymc as pm
+import pytensor
+import pytensor.tensor as pt
+import pytest
+import statsmodels.api as sm
+from numpy.testing import assert_allclose
+
+from pymc_experimental.statespace import BayesianVARMAX
+
+floatX = pytensor.config.floatX
+ROOT = Path(__file__).parent.absolute()
```

Comment:
Can you just use the long path `pymc_experimental/tests/.../*.csv`?

Reply:
I was having trouble with invalid file paths when I ran the tests via github actions. I can try it again using pymc's CI setup.